presto-docker-image/README.md

```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
 The default configuration uses `/data/presto` as the default for
 `node.data-dir`. Thus if using the default configuration and a mounted volume
 is desired for the data directory it should be mounted to `/data/presto`.
+
+### Quickstart
+The following steps will guide to quick test PrestoSQL using docker image.
+
+ 1. Run the PrestoSQL docker
+
+  To launch single machine prestosql instance for testing which will function as both a coordinator and worker, just run:
+
+  ```bash
+  docker run -p 8080:8080 prestosql/presto:latest
+  ```
+  
+  NOTE: You need to have [docker](https://docs.docker.com/install/) installed in the system
+
+  Wait for the message: `INFO	main	io.prestosql.server.PrestoServer	======== SERVER STARTED ========`
+
+  Now PrestoSQL server is running at `localhost:8080` or `YOUR-INSTANCE-IP:8080`
+ 
+ 2. Connect to PrestoSQL 
+ 
+  To connect to PrestoSQL server: 
+  
+  Download the PrestoSQL CLI (terminal-based interactive shell for running queries) using command: 
+  ```bash
+  wget -O presto-cli-312 https://repo1.maven.org/maven2/io/prestosql/presto-cli/312/presto-cli-312-executable.jar
+  ```
+  NOTE: [Java](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) should already be installed in system. Run `apt install openjdk-8-jre-headless` OR `brew install java` 
```

Comment:
When cli is prepackaged in the docker, this line becomes obsolete.

Reply:
CLI is already packaged.